### DISCUSSION TITLE
RestEasy Reactive query parameter list separator

### DISCUSSION THREAD

**Discussion by preokane** [quarkusio/quarkus#29500] [Q&A] [answered]
Similar to #27115 , I've been trying to get Resteasy Reactive to parse arrays of values in my URLs in the form of comma separated values rather than '&' separated values.
 e.g. **?array=item1,item2,item3** rather than **?array=item1&array=item2&array=item3**

This feature is available in your 'Resteasy Classic' extension through the usage of the @Separator annotation and the MultiValuedParamConverter which comes included in resteasy-core. 

It would look something like:

```
@GET
public String hello(@QueryParam("foo") @Separator(",") List<String> foo) {}
```
Why is this not included Resteasy Reactive?

P.S
In the meantime I've tried implementing my own List<?> converter and provider to do this, but I don't quite know how I would delegate the conversion of the list elements themselves to their respective Quarkus default converters, if such a thing is even possible. Any ideas?
P.P.S
#23609 seems to be relevant here

[a comment by an automated account is not shown]

**Comment by geoand** [comment, marked as answer]
Can you please open a github issue using the description you've added here?

I was not aware that RESTEasy Classic implements this feature, so we should defiinitely add it to RESTEasy Reactive.

**Reply by preokane** [reply]
Done. #29528
